package.json

```diff
@@ -31,75 +31,78 @@
     "ember-truth-helpers": "^3.1.1",
     "ion-rangeslider": "^2.3.1",
     "money-formatter": "^0.1.4"
   },
+  "peerDependencies": {
+    "qunit": "^2"
+  },
   "devDependencies": {
     "@babel/core": "^7.11.6",
     "@ember/jquery": "^2.0.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.8.1",
+    "@ember/test-helpers": "^3.2.1",
     "@embroider/test-setup": "^0.48.1",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
+    "@glint/template": "^1.2.1",
     "@storybook/addon-actions": "6.4.9",
     "@storybook/addon-essentials": "6.4.9",
     "@storybook/addon-links": "6.4.9",
     "@storybook/ember": "6.4.9",
     "@storybook/ember-cli-storybook": "^0.4.0",
-    "@types/ember-qunit": "^3.4.14",
-    "@types/ember-resolver": "^5.0.10",
-    "@types/ember__application": "^3.16.3",
-    "@types/ember__array": "^3.16.4",
-    "@types/ember__component": "^3.16.6",
-    "@types/ember__controller": "^3.16.5",
-    "@types/ember__debug": "^3.16.4",
-    "@types/ember__engine": "^3.16.3",
-    "@types/ember__error": "^3.16.1",
-    "@types/ember__object": "^3.12.6",
-    "@types/ember__polyfills": "^3.12.1",
-    "@types/ember__routing": "^3.16.15",
-    "@types/ember__runloop": "^3.16.3",
-    "@types/ember__service": "^3.16.1",
-    "@types/ember__string": "^3.16.3",
-    "@types/ember__template": "^3.16.1",
-    "@types/ember__test": "^3.16.1",
-    "@types/ember__test-helpers": "^2.0.1",
-    "@types/ember__utils": "^3.16.2",
-    "@types/htmlbars-inline-precompile": "^1.0.1",
-    "@types/jquery": "^3.5.6",
-    "@types/qunit": "^2.11.2",
-    "@types/rsvp": "^4.0.4",
+    "@tsconfig/ember": "^3.0.3",
+    "@types/ember": "^4.0.10",
+    "@types/ember-qunit": "^6.1.1",
+    "@types/ember-resolver": "^9.0.0",
+    "@types/ember__application": "^4.0.10",
+    "@types/ember__array": "^4.0.9",
+    "@types/ember__component": "^4.0.21",
+    "@types/ember__controller": "^4.0.11",
+    "@types/ember__debug": "^4.0.7",
+    "@types/ember__destroyable": "^4.0.4",
+    "@types/ember__engine": "^4.0.10",
+    "@types/ember__error": "^4.0.5",
+    "@types/ember__object": "^4.0.11",
+    "@types/ember__polyfills": "^4.0.5",
+    "@types/ember__routing": "^4.0.19",
+    "@types/ember__runloop": "^4.0.8",
+    "@types/ember__service": "^4.0.8",
+    "@types/ember__string": "^3.0.13",
+    "@types/ember__template": "^4.0.5",
+    "@types/ember__test": "^4.0.5",
+    "@types/ember__utils": "^4.0.6",
+    "@types/jquery": "^3.5.29",
+    "@types/qunit": "^2.19.9",
+    "@types/rsvp": "^4.0.8",
     "@types/sinon": "^10.0.2",
     "@typescript-eslint/parser": "^5.0.0",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.1.0",
-    "ember-auto-import": "^1.5.3",
     "ember-cli": "~3.28.6",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-typescript-blueprints": "^3.0.0",
+    "ember-cli-typescript-blueprints": "muziejus/ember-cli-typescript-blueprints#ember-4-update",
```

Comment:
are we sure about this?

Reply:
Forked and used our version for extra safety